iocore/net/BIO_fastopen.cc

```diff
@@ -116,39 +108,18 @@ static long
 fastopen_ctrl(BIO *bio, int cmd, long larg, void *ptr)
 {
   switch (cmd) {
-  case BIO_C_SET_FD:
-    ink_assert(larg == BIO_CLOSE || larg == BIO_NOCLOSE);
-    ink_assert(bio->num == NO_FD);
-
-    bio->init     = 1;
-    bio->shutdown = larg;
-    bio->num      = *reinterpret_cast<int *>(ptr);
-    return 0;
-
   case BIO_C_SET_CONNECT:
     // We only support BIO_set_conn_address(), which sets a sockaddr.
     ink_assert(larg == 2);
-    bio->ptr = ptr;
-    return 0;
-
-  // We are unbuffered so unconditionally succeed on BIO_flush().
-  case BIO_CTRL_FLUSH:
```

Comment:
Now this case is delegated to the ordinary socket BIO:

```C
return BIO_meth_get_ctrl(const_cast<BIO_METHOD *>(BIO_s_socket()))(bio, cmd, larg, ptr);
```

Do I understand this feedback right?